book/01-guided-tour.html

```diff
@@ -29,24 +29,39 @@ <h1>A Guided Tour</h1>
     toplevel should mostly work fine.</p>
 
     <div data-type="note">
-      <h1><code>Core</code>, <code>Core_kernel</code> and
-      portability</h1>
-
-      <p><code>Core_kernel</code> is actually part of a more
-      expansive library called <code>Core</code>. We use
-      <code>Core_kernel</code> here because it is portable to every
-      platform that OCaml runs on, unlike <code>Core</code>, which
-      only works on Unix-like platforms, including Linux and MacOS.
-      If you want your code to run on Windows or Javascript, you
-      need to stick to <code>Core_kernel</code>.</p>
-
-      <p>You should check out <code>Core</code> if you'll be running
-      only on Unix. <code>Core</code> includes a lot of useful
-      functionality, including an excellent command line parsing
-      library and modules that provide access to low-level operating
-      systems facilities from mutexes to sockets. We'll discuss some
-      of the additional functionality provided by <code>Core</code>
-      later in the book.</p>
+      <h1><code>Base</code>, <code>Core</code> and <code>Core_kernel</code></h1>
+
+      <p><code>Base</code> is one of a family of three standard library
+      replacements, each with different use-cases. Here's a quick
+	summary.
+
+	<ul>
+	  <li><em><code>Base</code></em> is the most minimal of the
+	    three.  It's designed to be portable, lightweight, and
+	    stable.
+	  </li>
+
+	  <li><em><code>Core_kernel<code></em>
+	    extends <code>Base</code> with many new data structures
+	    and capabilities. It's still portable, but has many more
+	    dependencies and takes longer to build, and also increases
+	    the size of your resulting executbale by more. Also, it
+	    contains some APIs that change faster than those
+	    in <code>Base</code>
+	  </li>
+
+	  <li><em><code>Core</code></em> is the most full-featured,
+	    extending <code>Core_kernel</code> with support for a variety of
+	    UNIX APIs, but only works on UNIX-like OSs, including Linux
+	    and Mac OS.
```

Comment:
Mac OS is now macOS.  We can fix this with a regexp globally

Reply:
Fixed.